README.md

```diff
@@ -1,4 +1,5 @@
 # GitVote
+[![CI](https://github.com/cncf/gitvote/actions/workflows/ci.yml/badge.svg)](https://github.com/cncf/gitvote/actions/workflows/ci.yml) ![GitHub release (latest SemVer)](https://img.shields.io/github/v/release/cncf/gitvote)
```

Comment:
Could you please replace the `release` badge with the `Artifact Hub` one? 🙂

[![Artifact Hub](https://img.shields.io/endpoint?url=https://artifacthub.io/badge/repository/gitvote)](https://artifacthub.io/packages/helm/gitvote/gitvote)

Reply:
No problem, done! 🙂